src/dawn/Optimizer/StencilInstantiation.h

```diff
@@ -163,6 +163,9 @@ class StencilInstantiation : NonCopyable {
   /// Set of all the IDs that are locally cached
   std::set<int> CachedVariableSet_;
 
+  /// table from FieldID to an Array of initialized dimensions
+  std::unordered_map<int, Array3i> fieldIDToInitializedDimensionsMap_;
```

Comment:
what is the meaning of `initialized`dimension here? it is not clear from the doc

Reply:
updated